### ISSUE TITLE
[BUG] "nan" Training loss in beginning of the training

### ISSUE THREAD

**Issue by purvang3** [rwightman/efficientdet-pytorch#269]
**Describe the bug**

**To Reproduce**
1. Download and unzip coco dataset
2. run following command
    " ./distributed_train.sh 2 ~/datasets/coco/ -b 8 --model tf_efficientdet_d0 --amp --lr .09 --sync-bn --model-ema --patience-epochs 5 --epochs 120 --aa COCO --output output/coco"

**Expected behavior**
Confusion: Do I explicitly need to convert bbox annotation format from xywh to yxyx?

**Screenshots**

<img width="1741" alt="Screen Shot 2022-03-08 at 1 17 15 PM" src="https://user-images.githubusercontent.com/39809304/157326954-f530ed8f-3919-47b6-882a-de655881cdc5.png">

Also, even after training for 100 epochs on Custom dataset, though training loss appears to be decreasing, coco eval stats doesn't improve.

<img width="1134" alt="Screen Shot 2022-03-08 at 1 25 31 PM" src="https://user-images.githubusercontent.com/39809304/157327773-84c2d63d-e8a6-4dbc-93df-9addb865f76d.png">

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Ubuntu 18.04]: Ubuntu 18.04, torch 1.8.1
 - This repository version [e.g. pypi 0.3.1 or branch/commit ref] :  latest clone
 - PyTorch version w/ CUDA/cuDNN  [e.g. from `conda list`, 1.7.0  py3.8_cuda11.0.221_cudnn8.0.3_0]
    installed conda env: cudatoolkit               10.2.89
                                       pytorch                     1.8.1

**Additional context**

**Comment by vladimircape** [comment]
How it was fixed?
I have same issue
```
Train: 0 [   0/199 (  0%)]  Loss: 21.143585 (21.1436)  Time: 2.179s,    0.46/s  (2.179s,    0.46/s)  LR: 1.000e-04  Data: 0.305 (0.305)
Train: 0 [  50/199 ( 25%)]  Loss:  2.364157 (2.3401)  Time: 0.267s,    3.75/s  (0.307s,    3.26/s)  LR: 1.000e-04  Data: 0.008 (0.014)
Train: 0 [ 100/199 ( 51%)]  Loss:  1.856368 (1.9741)  Time: 0.255s,    3.92/s  (0.288s,    3.47/s)  LR: 1.000e-04  Data: 0.008 (0.011)
Train: 0 [ 150/199 ( 76%)]  Loss:  1.307798 (1.7914)  Time: 0.290s,    3.45/s  (0.283s,    3.54/s)  LR: 1.000e-04  Data: 0.009 (0.010)
Train: 0 [ 198/199 (100%)]  Loss:  1.229478 (1.6821)  Time: 0.311s,    3.22/s  (0.280s,    3.57/s)  LR: 1.000e-04  Data: 0.060 (0.010)
Test: [   0/28]  Time: 0.360 (0.360)  Loss:  1.1886 (1.1886)  
Test: [  28/28]  Time: 0.120 (0.092)  Loss:  1.1273 (1.2172)  
Metrics:
PascalBoxes_Precision/mAP@0.5IOU: 0.035370736627820726
PascalBoxes_PerformanceByCategory/AP@0.5IOU/hand: 0.035370736627820726
Current checkpoints:
 ('./output/train/20220630-161049-efficientdetv2_ds/checkpoint-0.pth.tar', 0.035370736627820726)

Train: 1 [   0/199 (  0%)]  Loss:  1.427459 (1.4275)  Time: 0.661s,    1.51/s  (0.661s,    1.51/s)  LR: 2.008e-02  Data: 0.307 (0.307)
Train: 1 [  50/199 ( 25%)]  Loss:  1.936068 (1.5579)  Time: 0.287s,    3.48/s  (0.294s,    3.41/s)  LR: 2.008e-02  Data: 0.008 (0.014)
Train: 1 [ 100/199 ( 51%)]  Loss:  1.335602 (1.6108)  Time: 0.312s,    3.21/s  (0.291s,    3.43/s)  LR: 2.008e-02  Data: 0.007 (0.012)
Train: 1 [ 150/199 ( 76%)]  Loss:  1.680240 (1.6052)  Time: 0.292s,    3.42/s  (0.291s,    3.44/s)  LR: 2.008e-02  Data: 0.007 (0.011)
Train: 1 [ 198/199 (100%)]  Loss:  1.641265 (1.5630)  Time: 0.345s,    2.90/s  (0.291s,    3.44/s)  LR: 2.008e-02  Data: 0.062 (0.010)
Test: [   0/28]  Time: 0.342 (0.342)  Loss:  3.9760 (3.9760)  
Test: [  28/28]  Time: 0.130 (0.087)  Loss:  5.5147 (3.9495)  
Metrics:
PascalBoxes_Precision/mAP@0.5IOU: 0.0
PascalBoxes_PerformanceByCategory/AP@0.5IOU/hand: 0.0
Current checkpoints:
 ('./output/train/20220630-161049-efficientdetv2_ds/checkpoint-0.pth.tar', 0.035370736627820726)
 ('./output/train/20220630-161049-efficientdetv2_ds/checkpoint-1.pth.tar', 0.0)

Train: 2 [   0/199 (  0%)]  Loss:  1.664125 (1.6641)  Time: 0.635s,    1.57/s  (0.635s,    1.57/s)  LR: 4.006e-02  Data: 0.297 (0.297)
Train: 2 [  50/199 ( 25%)]  Loss:  2.365582 (1.7575)  Time: 0.307s,    3.26/s  (0.297s,    3.37/s)  LR: 4.006e-02  Data: 0.007 (0.014)
Train: 2 [ 100/199 ( 51%)]  Loss:       nan (   nan)  Time: 0.285s,    3.51/s  (0.293s,    3.41/s)  LR: 4.006e-02  Data: 0.007 (0.012)
Train: 2 [ 150/199 ( 76%)]  Loss:       nan (   nan)  Time: 0.289s,    3.46/s  (0.292s,    3.42/s)  LR: 4.006e-02  Data: 0.008 (0.011)
Train: 2 [ 198/199 (100%)]  Loss:       nan (   nan)  Time: 0.350s,    2.85/s  (0.293s,    3.42/s)  LR: 4.006e-02  Data: 0.067 (0.011)
Test: [   0/28]  Time: 0.425 (0.425)  Loss:     nan (   nan)  
Test: [  28/28]  Time: 0.148 (0.090)  Loss:     nan (   nan)  
Metrics:
PascalBoxes_Precision/mAP@0.5IOU: 0.0
PascalBoxes_PerformanceByCategory/AP@0.5IOU/hand: 0.0
Current checkpoints:
 ('./output/train/20220630-161049-efficientdetv2_ds/checkpoint-0.pth.tar', 0.035370736627820726)
 ('./output/train/20220630-161049-efficientdetv2_ds/checkpoint-1.pth.tar', 0.0)
 ('./output/train/20220630-161049-efficientdetv2_ds/checkpoint-2.pth.tar', 0.0)


```
